docs/src/main/sphinx/admin/fault-tolerant-execution.rst

```diff
@@ -536,6 +546,20 @@ GCS bucket as the spooling storage destination.
     exchange.s3.endpoint=https://storage.googleapis.com
     exchange.gcs.json-key-file-path=/path/to/gcs_keyfile.json
 
+.. _fte-exchange-hdfs:
+
+HDFS
+~~~~~~~~~~~~~~~~~~
```

Comment:
Shorten this to be same length as title itself

Reply:
Done